12/umbraco-cms/tutorials/getting-started-with-entity-framework-core.md

```diff
@@ -0,0 +1,226 @@
+﻿---
+versionFrom: 12.0.0
+meta.Title: Umbraco Database
+description: A guide to getting started with Entity Framework Core in Umbraco
+---
+
+# Getting started with Entity Framework Core in Umbraco
+
+Using EFCore, it is possible to add custom database tables to your site to store additional data that should not be stored as normal content nodes.
+
+The end result looks like this:
+
+![Database result of a migration](images/db-table.png)
+
+We can then work with this data through Entity framework.
```

Comment:
Just to be sure, this is after one is done with the tutorial, then the data can be worked with through the entity framework?

Reply:
Yes 👍Essentially you end up with some working code samples of how it could be done at least 👍

Reply:
Great, thanks for elaborating :D